identity/src/main/java/com/stripe/android/identity/IdentityActivity.kt

```diff
@@ -378,7 +322,7 @@ internal class IdentityActivity :
                 return
             }
             when {
-                isConsentFragment(destination) -> {
+                isConsent(destination) -> {
```

Comment:
minor - maybe we can now just do `when(destination) { ... ConfirmationDestination.ROUTE.route -> ... }`